ftw/redirector/browser/configure.zcml

```diff
@@ -15,4 +15,20 @@
         layer="ftw.redirector.interfaces.IRedirectorLayer"
         />
 
+    <browser:page
+        name="export"
+        for="..interfaces.IRedirectConfig"
+        class=".excel_views.ExportRedirectConfigView"
+        permission="zope2.View"
+        layer="ftw.redirector.interfaces.IRedirectorLayer"
```

Comment:
The `layer` is not required here, since we are not customizing an existing view and the view is bound to a type of ours anyway (`for`).